Visage.FrontEnd/Visage.FrontEnd.Shared/Styles/input.css

```
@@ -1,3 +1,23 @@
-﻿@tailwind base;
-@tailwind components;
-@tailwind utilities;
+﻿@import "tailwindcss";
+@plugin "@daisyui" {
+    themes: light --default, dark --prefersdark;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)